.github/workflows/node.js.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
We need to keep supporting Node v14 for supporting Fastify v4. Just exclude Windows v14 which is broken on GHA.

Reply:
we still suuport v14? 🤦🏻

Reply:
we need to plan fastify v5